Formula/hledger.rb

```diff
@@ -3,7 +3,7 @@ class Hledger < Formula
   homepage "https://hledger.org/"
   url "https://hackage.haskell.org/package/hledger-1.18.1/hledger-1.18.1.tar.gz"
   sha256 "0c88c9a1896a6c431854c76229f0fe8d9bc59b6560829a4af1b8fcbad85486fa"
-  license "GPL-3.0"
+  license "GPL-3.0-only"
```

Comment:
```suggestion
  license "GPL-3.0-or-later"
```

https://hledger.org/README.html

>  hledger is free software available under the GNU General Public License v3+.